environments/india/inventory.ini

```diff
@@ -88,6 +88,12 @@ hot_standby_master=10.203.10.190
 [proxy:children]
 proxy1
 
+[filebeat:children]
```

Comment:
Instead of creating a new inventory group for this what about just using the existing groups since we know where we want filebeat to run already?

Reply:
Make sense.